planet/src/main/webapp/js/profile.js

```diff
@@ -0,0 +1,143 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+function diplaySection(sectionId) {
+    switch(sectionId) {
+        case 'about-section':
+            document.getElementById('about-section').style.display = 'block';
+            document.getElementById('posts-section').style.display = 'none';
+            document.getElementById('events-section').style.display = 'none';
+            break;
+        case 'posts-section':
+            document.getElementById('about-section').style.display = 'none';
+            document.getElementById('posts-section').style.display = 'block';
+            document.getElementById('events-section').style.display = 'none';
+            break;
+        case 'events-section':
+            document.getElementById('about-section').style.display = 'none';
+            document.getElementById('posts-section').style.display = 'none';
+            document.getElementById('events-section').style.display = 'block';
+            break;
```

Comment:
Add a default block for some error reporting (e.g. "ERROR: unhandled section ID " + sectionId)

Reply:
Fixed